### DISCUSSION TITLE
Center a TitleStyle ?

### DISCUSSION THREAD

**Discussion by Benoite142** [py-pdf/fpdf2#1280] [Q&A]
Hi,

I've been trying to center a TitleStyle for quite some time now. I've been using a TitleStyle for the `start_section` function used in the TOC placeholder and I am having a lot of trouble trying to center the TitleStyle. I know we can easily center some cell or text with `align='C'` but I found no way of doing that for the TitleStyle. I know we can put a value to the `l_margin` to make it look like it is centered, but I can't really use that since all of my titles differs in length. 

I also tried aligning everything to the center of the page, but that still doesn't work with TitleStyle and the sections.
Worst case scenario is that I put everything on the left, but I would really want it to be centered.

If anyone knows how I could do this, please let me know.

Thank you for you time 😊

**Comment by Lucas-C** [comment]
Hi @Benoite142

Thank you for reaching out.

First of all, `TitleStyle` has been deprecated and renamed into `TextStyle` in the latest version (`2.8.1`).

Now, you are entirely right that this currently not supported by `fpdf2`, bu this is certainly something that we could!

I am converting this discussion into a feature request issue in order for a benevolant developper to be able to implement this: https://github.com/py-pdf/fpdf2/issues/1282

**Reply by Benoite142** [reply]
Thanks for the response,

Oh! I didn't know that `TitleStyle` was depracated so thank you for the info.

Ok, hopefully it won't be too hard to implement!

Thanks again!
